src/charts/onResize/drawNormalRange.js

```diff
@@ -0,0 +1,17 @@
+export default function drawNormalRange() {
+    this.svg.select('.normal-range').remove();
+    this.svg
+        .insert('rect', '.line-supergroup')
+        .classed('normal-range', true)
+        .attr({
```

Comment:
I'd add `.classed("hidden", this.config.normal_range_method === 'none')`

Reply:
Good idea.